sections/designRules.md

```diff
@@ -9,6 +9,7 @@
 The REST architectural style is centered around the concept of a [=resource=]. A resource is an abstraction of a conceptual entity, identified by a globally unique [=URI=]. It may correspond to anything from a physical object (e.g. a building or a person) to an abstract concept (e.g. a permit, an event or today's weather). Although a resource is not tied to any specific exchange format, its current state can be transferred to clients through one or more representations, such as JSON or XML.
 
 <span id="api-05"></span>
+
```

Comment:
Stylistische wijzigingen moeten worden gescheiden van inhoudelijke wijzigingen. Stylistische wijzigingen kunnen we namelijk elk moment mergen, inhoudelijke wijzigingen enkel na goedkeuring bij Technisch Overleg.

Dat gezegd hebbende, deze new lines lijken mij niet nodig, dus die zou ik terug draaien. De wijzigingen aan lijsten kan wel, dus stuur daar gerust een aparte PR voor.